.github/actions/upload/examples/README.md

```diff
@@ -0,0 +1,204 @@
+# Workflow Examples
```

Comment:
I think this readme needs some fresh eyes, it feels like it might have mutated a few times and lost a little coherence? It could be a fair bit shorter too I think.